README.md

```diff
@@ -1,3 +1,33 @@
 # REDCap to Harvest
 
 Dockerfile and tools for deriving Harvest applications from REDCap projects.
+
+Installation and Setup
+----------------------
+
+This tool runs using Docker. Install it if you don't have it:
+
+[Install Docker](http://docs.docker.com/installation/) 
+
+In order to run the container, you need to place your data dictonary and data file in a folder called Input. Name the data dictonary and data file "metadata.csv" and "data.csv", respectively.
+
+Now run the container:
+
+```bash
+docker run --rm -it -p 8001:8000 -v </Path/To/Your/InputFolder>:/input dbhi/redcap-harvest
```

Comment:
Remove angle brackets and lowercase `/Path/To...`. No need for two visual cues that this is user-defined.